autoload/lightline/colorscheme/materia.vim

```diff
@@ -0,0 +1,63 @@
+" =============================================================================
+" Filename: autoload/lightline/colorscheme/materia.vim
+" Author: Lokesh Krishna
+" License: MIT License
+" Last Change: 2017/10/21 11:32:27.
+" =============================================================================
+
+" Common colors
+let s:fg     = '#d5dbe5'
+let s:blue   = '#89ddff'
+let s:green  = '#8bd649'
+let s:purple = '#82aaff'
+let s:red1   = '#ec5f67'
+let s:red2   = '#ec5f67'
+let s:yellow = '#ffcc00'
+
+let s:p = {'normal': {}, 'inactive': {}, 'insert': {}, 'replace': {}, 'visual': {}, 'tabline': {}}
+
+if &background ==# 'light'
+  " Light variant
+  let s:bg     = '#ffffff'
+  let s:gray1  = '#2c393f'
+  let s:gray2  = '#d5dbe5'
+  let s:gray3  = '#707880'
+
+  let s:p.normal.left     = [ [ s:bg, s:green, 'bold' ], [ s:gray1, s:gray3 ] ]
```

Comment:
It's not a strict convention but I prefer blue for normal and green for insert.

Reply:
Just noticed that that is true for almost all of the offered colorschemes but I edited the one theme and the colors are swapped in it :P 

I've swapped them back now :)